schema/appmodel/application.schema.xml

```diff
@@ -294,6 +295,10 @@
   <attribute name="preallocation" type="bool" init-value="true"/>
  </class>
 
+ <class name="LatencyMonitoringConf">
+  <attribute name="enable_latency_monitoring" type="bool" init-value="true"/>
+ </class>
```

Comment:
Personally I think an entire new class to overlay a boolean is a bit of an overkill.

On one hand being able to switch all the monitoring on with one object is nice, but on the other hand, it adds extra complexity when we want to switch only one of these on. And I imagine e.g. by default we might it off, say, `RawDataProcessor` and `TPDataProcessor` (especially since now it's on per-TP, not per-TPSet basis) but on in the `TCDataProcessor` and in the `MLT`. It could also be added in `DataProcessor`, so it doesn't need to be in all the derived processors separately.

Perhaps @glehmannmiotto has a more elegant solution here? Perhaps enabling latency monitoring via application? There are other non-`DataProcessor`-derived objects that also need this.

Reply:
I agree with your assessment @ArturSztuc, and think that we should only add a boolean attribute. The question for @MRiganSUSX is who is in charge of the monitoring. Is it the application, the module or the data processor? Depending on this we should put the attribute in the right class (DataProcessor may be the right one). If you are worried about the ease of enabling/disabling this monitoring feature (although, why would you ever want to disable it?), then it is easy to add a python utility in daqconf that changes the value of the attribute for an application, a class of applications, or all.